SDK/AppCenterData/Microsoft.AppCenter.Data.iOS/RemoteOperationDelegate.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AppCenter.Data.iOS.Bindings;
+using System;
+
+namespace Microsoft.AppCenter.Data.iOS
+{
+    public class RemoteOperationDelegate : MSRemoteOperationDelegate
+    {
+        public override void DataDidCompletedRemoteOperation(MSData data, string operation, MSDocumentMetadata documentMetadata, MSDataError error)
+        {
+            OnDataDidCompletedRemoteOperationAction.Invoke(operation, documentMetadata, error);
+        }
+
+        public Action<string, MSDocumentMetadata, MSDataError> OnDataDidCompletedRemoteOperationAction { get; set; }
```

Comment:
If that was an inner class of Data like Android you wouldn't need the action to forward

Reply:
it's good practice to keep them split, this also aligns with the Objective-C implementation